src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeFeature.cs

```diff
@@ -18,6 +18,11 @@ public static partial class RuntimeFeature
         public const string DefaultImplementationsOfInterfaces = nameof(DefaultImplementationsOfInterfaces);
 #endif
 
+        /// <summary>
+        /// Indicates that this version of runtime supports the Unmanaged calling convention value.
+        /// </summary>
+        public const string UnmanagedSignatureCallingConvention = nameof(UnmanagedSignatureCallingConvention);
```

Comment:
Doesn't this need to be added to the switch below?

Reply:
I honestly thought the presence of the string was what was needed. Didn't realize how the system worked.

Reply:
The comment "Indicates that this version of runtime supports" was the confusing part. The string constant doesn't indicate anything, it is simply the name of the feature.

@lambdageek I was going to make this the default for SPCL. Would you like me to mark this as `false` for mono until support can be added?

Reply:
> I honestly thought the presence of the string was what was needed. Didn't realize how the system worked.

Well, the compiler will just look for the presence of the member. But I assumed that since everything else was in the switch, it should be consistent.

Reply:
Actually the tests don't really permit making this constant dynamic so it looks like for the time being mono will indicate it is supported, but not. Unless I update the following test:

https://github.com/dotnet/runtime/blob/106cd9ebaed5f974fa8f17e153ec3e22b6576f43/src/libraries/System.Runtime/tests/System/Runtime/CompilerServices/RuntimeFeatureTests.cs#L41-L57

Reply:
The current API doesn't support dynamic support so I am going to update this to lie about present mono with the expectation that at GA it will be truthful about support in mono.

/cc @lambdageek